Consul.Test/OperatorTest.cs

```diff
@@ -17,13 +17,21 @@
 //  </copyright>
 // -----------------------------------------------------------------------
 
+using System;
 using System.Threading.Tasks;
 using Xunit;
 
 namespace Consul.Test
 {
     public class OperatorTest : BaseFixture
     {
+        private static readonly Random Random = new Random();
+        internal static string GeneratePeerDatacenterName()
+        {
+            int randomNumber = Random.Next(100, 1000);
+            return $"ran-dc-{randomNumber}";
```

Comment:
Single random number might be not enough to prevent from conflicts/duplicates. We've already a method `KVTest.GenerateTestKeyName()` which is being used in many places in tests already. It generates 16 character long random string. Please use it instead.

Reply:
@marcin-krystianc thanks, you can take another look now.